spec/eventbrite_sdk_spec.rb

```diff
@@ -126,6 +146,34 @@
       end
     end
 
+    context 'with request token' do
+      it 'sets the Authorization header with the given api token' do
+        token = 'request_token'
+        described_class.token = token
```

Comment:
This test adds the value 'request_token' to the module token and it also passes the value as the request token.

I think it would be a better test if a different value was set as the module token, that way it's clear the value set in the header is the request token - and it shows the request token takes precedence over the module token.